src/GlobalAccountHeader/GlobalAccountHeader.scss

```diff
@@ -0,0 +1,41 @@
+.global-header-account {
+  align-items: center;
+  display: none;
+  white-space: nowrap;
+
+  a {
+    font-family: $montserratSemiBoldFamily;
+    font-size: 1.2rem;
+    white-space: nowrap;
+
+    @include ms-ie-10-11-only {
+      font-family: $montserratFamily;
+    }
+
+    @include ms-edge-only {
+      font-family: $montserratFamily;
+    }
+  }
+
+  .header-email-capture__header {
+    letter-spacing: 0.09rem;
+  }
+}
+
+.global-header-account__link {
+  font-family: $montserratFamily;
+  font-size: $font-size-small;
+  padding-right: 2rem;
+
+  &.button.tomato {
```

Comment:
might have to figure out another approach to handle the specificity issue this `&.button.tomato` rule is solving.